mlir/include/mlir/Dialect/Rock/IR/RockAttrDefs.td

```diff
@@ -50,23 +50,23 @@ class Rock_Attr<string name, list<Trait> traits = []> :
     AttrDef<Rock_Dialect, name, traits>;
 
 /// ConvOpType
-def ConvOp2DType : I32EnumAttrCase<"Fwd", 0, "conv2d">;
-def ConvOp2DBwdDataType : I32EnumAttrCase<"BwdData", 1, "conv2d_bwd_data">;
-def ConvOp2DBwdWeightType : I32EnumAttrCase<"BwdWeight", 2, "conv2d_bwd_weight">;
+def ConvOpType : I32EnumAttrCase<"Fwd", 0, "conv">;
```

Comment:
The heart of the conv2d-to-conv name change.  I could locate most of the ripples by making this change, compiling, and seeing what broke.